### PR TITLE
eval: LLM based response validation

### PR DESCRIPTION
**Pull request by tisnik**
## Description

eval: LLM based response validation

## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [ ] CI configuration change
- [ ] Konflux configuration change

### CODE REVIEW

File: scripts/evaluation/README.md [hunk 1]
```diff
@@ -47,7 +47,9 @@ Please refer above files for the structure, add new data accordingly.
 **eval_out_dir**: Directory, where output csv/json files will be saved.
 
 **eval_metrics**: By default all scores/metrics are calculated, but this decides which scores will be used to create the graph.
-This is a list of metrics. Ex: cosine, euclidean distance, precision/recall/F1 score.
+This is a list of metrics. Ex: cosine, euclidean distance, precision/recall/F1 score, answer relevancy score, LLM based similarity score.
+
+**judge_provider / judge_model**: Provider / Model for judge LLM. This is required for LLM based evaluation (answer relevancy score, LLM based similarity score). This needs to be configured correctly through config yaml file. [Sample provider/model configuration](../../examples/olsconfig.yaml)
```
Comment (r0x0d) [inline scripts/evaluation/README.md:52]:
```suggestion
**judge_provider / judge_model**: Provider / Model for judge LLM. This is required for LLM based evaluation (answer relevancy score, LLM based similarity score). This needs to be configured correctly through config yaml file. [Sample provider/model configuration](../../examples/rcsconfig.yaml)
```

Comment (tisnik) [reply scripts/evaluation/README.md:52]: indeed @r0x0d , lemme fix it

Comment (tisnik) [reply scripts/evaluation/README.md]: fixed @r0x0d

**Pull request closed without merging**